packages/gamut/src/Tabs/Tab/index.tsx

```diff
@@ -26,6 +26,7 @@ export const Tab: FunctionComponent<TabProps> = ({
 }: TabProps) => {
   const tabLinkClasses = cx(s.tab, {
     [s.active]: active,
+    [className]: className !== undefined,
     [activeClassName]: active && activeClassName !== undefined,
   });
```

Comment:
`classnames` will remove blank/`null`/`undefined` class names for us.

```suggestion
  const tabLinkClasses = cx(s.tab, className, {
    [s.active]: active,
    [activeClassName]: active && activeClassName !== undefined,
  });
```